doc/Changelog.md

```diff
@@ -3,6 +3,21 @@ Changelog {#Changelog}
 
 ## Rendering Resource Manager
 
+# Release 0.4.0
```

Comment:
Yes, I need another PR for changing the version. The build mechanism forces a dedicated PR for changing the version number.